abi/vfabia64/vfabia64.rst

```diff
@@ -1489,6 +1489,13 @@ Masked signature generation for ``[not]inbranch`` clause.
 |``#pragma omp declare simd notinbranch`` |No     |Yes     |Yes   |No      |
 +-----------------------------------------+-------+--------+------+--------+
 
+Inactive lanes
+--------------
+
+The values of the inactive lanes of the input parameters and the
```

Comment:
Sorry to nit-pick, but maybe:

In a masked vector function, the contents of the inactive lanes of the input parameters and the inactive lanes of the return value are undefined.

Reply:
Done